lib/Exporter/FilesExporter.php

```diff
@@ -51,7 +51,7 @@ public function export(string $userId, string $exportPath) {
 		$userFolder = $this->rootFolder->getUserFolder($userId)->getParent();
 
 		/** @var \OCP\Files\Node $node */
-		foreach (new UserStorageIterator($userFolder) as $node) {
+		foreach (RecursiveNodeIterator::create($userFolder) as $node) {
```

Comment:
I don't think this provides any advantage. We need a factory to be injected to be able to mock the iterator.